src/_includes/macros/activity-snippets.html

```diff
@@ -13,7 +13,7 @@
 
    activity_types:    An array of activity types.
 
-                      Possible values: posts, op-ed, press release,
+                      Possible values: posts, op-ed, newsroom, press release,
```

Comment:
Oh wait, this changed in #782. Let's get that merged before this so we have a better diff